metrics/android-render-test-runner/render-tests/map-mode/tile-avoid-edges/metrics.json

```diff
@@ -2,8 +2,8 @@
     "network": [
         [
             "probeNetwork - default - end",
-            12,
-            340753
+            8,
```

Comment:
I'm curious: why did these change with this PR? I would have though that this PR would not change any results and it would only fix a rare race (that I presume was not previously impacting the baselines).